templates/core/terraform/network/locals.tf

```diff
@@ -1,12 +1,14 @@
 locals {
-  core_services_vnet_subnets               = cidrsubnets(var.core_address_space, 4, 4, 4, 4, 2, 2, 2)
-  firewall_subnet_address_space            = local.core_services_vnet_subnets[0] # .0 - .62
+  core_services_vnet_subnets               = cidrsubnets(var.core_address_space, 4, 4, 4, 4, 2, 2, 4, 4, 4)
+  firewall_subnet_address_space            = local.core_services_vnet_subnets[0] # .0 - .63
   app_gw_subnet_address_prefix             = local.core_services_vnet_subnets[1] # .64 - .127
   bastion_subnet_address_prefix            = local.core_services_vnet_subnets[2] # .128 - .191
   web_app_subnet_address_prefix            = local.core_services_vnet_subnets[3] # .192 - .254
   shared_services_subnet_address_prefix    = local.core_services_vnet_subnets[4] # .0 - .254
-  aci_subnet_address_prefix                = local.core_services_vnet_subnets[5] # .0 - .254
-  resource_processor_subnet_address_prefix = local.core_services_vnet_subnets[6] # .0 - .254
+  resource_processor_subnet_address_prefix = local.core_services_vnet_subnets[5] # .0 - .254
```

Comment:
How does this work with the upgrade of an existing implementation? Is the resource processor subnet changing?

Reply:
I think the network ranges change, but I removed the unused aci

Reply:
Subnet address ranges cannot be changed once created I don't believe. Can you verify this as we need to be able to upgrade existing deployments.

Reply:
I will reuse the aci

Reply:
fixed.
before:
<img width="1778" alt="1" src="https://user-images.githubusercontent.com/13205761/173915041-c9dfc0a2-eebc-4256-b76b-53dd92369f53.png">
after:
<img width="1778" alt="2" src="https://user-images.githubusercontent.com/13205761/173915111-be08050f-399c-4ff9-9c6a-a6226edf83e9.png">